### PR TITLE
feat: bors

### PR DESCRIPTION
**Pull request by EstebanBorai**

### CODE REVIEW

**Comment by morenol** [comment]
bors ping

**Comment by morenol** [comment]
@sehz could you enable bors on this repo?

**Comment by sehz** [comment]
bors was already installed org wide.  I added http, mqtt, Kafka and SQL connectors

**Comment by EstebanBorai** [comment]
@morenol perhaps this get activated merging? So the `bors.toml` can be fetched?

**Comment by morenol** [comment]
bors ping

**Review by morenol — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,7 +7,9 @@ concurrency:
 on:
   workflow_dispatch:
   pull_request:
```
Comment (morenol) [inline .github/workflows/ci.yml:9]:
```suggestion
  push:
    branches:
      - staging
      - trying
  pull_request:
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -7,7 +7,9 @@ concurrency:
...
-    branches: [main]
+    branches:
+      - staging
+      - trying
```
Comment (morenol) [inline .github/workflows/ci.yml:12]:
```suggestion
    branches: [main]
```

**Pull request merged**